includes/Core/Modules/Modules.php

```diff
@@ -806,68 +807,71 @@ private function prepare_module_data_for_response( Module $module ) {
 	 * Gets the REST schema for a module.
 	 *
 	 * @since 1.3.0
+	 * @since n.e.x.t Returns a function that returns the schema when invoked.
 	 *
-	 * @return array Module REST schema.
+	 * @return Closure Function which returns the Module REST schema.
```

Comment:
This works, but why not keep this method as is and pass a closure that calls this to WP instead? I think it's more intuitive for a method called `get_module_schema` to actually return the schema directly.

Reply:
Yeah that makes more sense, I probably misunderstood the IB. I'll update it to be this way instead 👍